docs/src/api/class-browsercontext.md

```diff
@@ -963,9 +963,14 @@ specified.
 * since: v1.8
 - `permissions` <[Array]<[string]>>
 
-A permission or an array of permissions to grant. Permissions can be one of the following values:
+A list of permissions to grant.
+
+:::danger
+Supported permissions differ between browsers, and even between different versions of the same browser. Any permission may stop working after an update.
```

Comment:
Last time I tried to give them a workaround with `newCDPSession` in order to grant custom permissions - for some reason it didn't work for me. Maybe it would make sense to give it to them aka. put it into the issue?

Reply:
I don't think we are interested in this usecase, but if someone files an issue about it, feel free to respond.